src/Microsoft.OData.Edm/Validation/ValidationRules.cs

```diff
@@ -2497,6 +2497,22 @@ public static class ValidationRules
                     }
                 });
 
+        /// <summary>
+        /// Validates that if a vocabulary annotation doesn't have a value, it should declare a type with a default value.
+        /// </summary>
+        public static readonly ValidationRule<IEdmVocabularyAnnotationWithDefault> VocabularyAnnotationUseDefaultTermMustHaveDefaultValue =
+            new ValidationRule<IEdmVocabularyAnnotationWithDefault>(
+                (context, annotation) =>
+                {
+                    if (annotation.UsesDefault && annotation.Term.DefaultValue == null)
+                    {
+                        context.AddError(
+                            annotation.Location(),
+                            EdmErrorCode.ExpressionMissingValueOrTermWithDefaultValue,
+                            "Annotation expressions must specify a value or use a term with a specified default value.");

```

Comment:
>Annotation expressions must specify a value or use a term with a specified default value. [](http://example.com/codeflow?start=29&length=89)

Move the error message into the .txt file.

Reply:
Done ✅